src/main/resources/templates/user-form.html

```diff
@@ -0,0 +1,12 @@
+<form>
+    <input placeholder="Id" name="userId"/>
+    <br>
+    <input placeholder="Name" name="userName"/>
+    <br>
+    <input placeholder="PhoneNumber" name="userPhoneNumber"/>
+    <br>
+    <button type="submit>"> save</button>
+    <br>
+    <button type="sybmit>"> delete</button>
```

Comment:
type="sybmit>">
doesn't IDE highlight that?

Reply:
You are right.